core/bin/oauth_generate_key.rs

```diff
@@ -0,0 +1,11 @@
+use base64::{engine::general_purpose, Engine as _};
```

Comment:
So the idea, it to use this script once to generate the encryption key before setting it as an env variable, is that right?

Reply:
Yeah can be removed once set in dev and prod